composer/datasets/hparams.py

```diff
@@ -102,7 +108,7 @@ class WebDatasetHparams(DatasetHparams, abc.ABC, metaclass=metaclass):
     def initialize_object(self, batch_size: int, dataloader_hparams: DataloaderHparams) -> Union[DataLoader, DataSpec]:
         """Creates a :class:`DataLoader` or :class:`DataloaderSpec` for this dataset.
 
-        Parameters:
+        Args:
```

Comment:
This text is repeated twice in the output. So may be remove it so it gets it from the base class.
![Screen Shot 2022-03-10 at 9 15 53 AM](https://user-images.githubusercontent.com/37562707/157718568-ca10ad16-740b-4668-a4d7-87955142d6b7.png)